windows/installer-source/KaliteSetupScript.iss

```diff
@@ -417,6 +417,38 @@ begin
     end;
 end;
 
+{ Used in GetPipPath below }
+const
+    DEFAULT_PATH = '\Python27\Scripts\pip.exe';
+
+{ Returns the path of pip.exe on the system. }
+{ Tries several different locations before prompting user. }
+function GetPipPath: string;
+var
+    path : string;
+    i : integer;
+begin
+    for i := Ord('C') to Ord('H') do

```

Comment:
But at any rate such a user will be prompted to select the location of pip.exe later on.